From: OpenWrt ER-12 project
Subject: [PATCH] octeon: add EdgeRouter 12 device definition

Add the Ubiquiti EdgeRouter 12 (ubnt_edgerouter-12) to the octeon
image Makefile. Inherits from the E300 base device, adds bonding,
802.1Q, ip-full and ip-bridge packages required by the er12-fabric
init script.

--- a/target/linux/octeon/image/Makefile
+++ b/target/linux/octeon/image/Makefile
@@ -75,6 +75,14 @@ define Device/ubnt_edgerouter-6p
 endef
 TARGET_DEVICES += ubnt_edgerouter-6p
 
+define Device/ubnt_edgerouter-12
+  $(Device/ubnt_edgerouter-e300)
+  DEVICE_MODEL := EdgeRouter 12
+  DEVICE_DTS := cn7130_ubnt_edgerouter-12
+  DEVICE_PACKAGES += kmod-bonding kmod-8021q ip-full ip-bridge
+endef
+TARGET_DEVICES += ubnt_edgerouter-12
+
 ERLITE_CMDLINE:=-mtdparts=phys_mapped_flash:512k(boot0)ro,512k(boot1)ro,64k(eeprom)ro root=/dev/sda2 rootfstype=squashfs,ext4 rootwait
 define Device/ubnt_edgerouter-lite
   DEVICE_VENDOR := Ubiquiti
